src/functions/get/handler.ts

```diff
@@ -0,0 +1,40 @@
+/**
+ * Copyright (c) Hathor Labs and its affiliates.
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import type { IValidatedEventAPIGatewayProxyEvent } from '@libs/api-gateway';
+import { formatJSONResponse } from '@libs/api-gateway';
+import { wrapWithConnection } from '@libs/lambda';
+
+import { validatePassword } from '@libs/util';
+import { LambdaError } from '@libs/errors';
+import { AUTHPASSWORD_HEADER_KEY } from '@libs/constants';
+import { getProposalFromDb } from '@services/proposals';
+
+const getProposal: IValidatedEventAPIGatewayProxyEvent<Object> = async (event) => {
+  const authPassword = event.headers[AUTHPASSWORD_HEADER_KEY] || '';
+  if (authPassword.length < 3) {
+    throw new LambdaError('Invalid password', 'INVALID_PASSWORD');
```

Comment:
Why don't we use an enum for the `ApiError` instead of a type so we can do something like `ApiError.INVALID_PASSWORD` like we do in the wallet-service?

Reply:
I've read plenty of articles recommending the use of `const types` instead of `enums` and, since I was having trouble with the correct typing on `src/libs/lambda.ts` I decided to give this recommendation a try.

Const enums also help the code completion suggest you the correct values just like with typing `ApiError.`, but I feel the final code is less readable.

After all, I didn't really like the results, so your suggestion is welcome. Refactored on [374980b](https://github.com/HathorNetwork/hathor-atomic-swap-service/pull/3/commits/374980bf028233f860554303ab34fcd5376cf92c).